validation/src/test/scala/hmda/validation/rules/ts/syntactical/S100Spec.scala

```diff
@@ -0,0 +1,50 @@
+package hmda.validation.rules.ts.syntactical
+
+import hmda.model.fi.lar._
+import hmda.model.fi.ts.{ Contact, Parent, Respondent, TransmittalSheet }
+import hmda.model.institution.Agency.CFPB
+import hmda.model.institution.ExternalIdType.{ FdicCertNo, FederalTaxId, RssdId }
+import hmda.model.institution.InstitutionStatus.Active
+import hmda.model.institution.InstitutionType.Bank
+import hmda.model.institution.{ ExternalId, Institution }
+import hmda.validation.context.ValidationContext
+import hmda.validation.dsl.{ Failure, Success }
+import org.scalatest.{ MustMatchers, WordSpec }
+
+class S100Spec extends WordSpec with MustMatchers {
```

Comment:
I think it might be useful to also include a test for: `ValidationContext(None, None)`